src/components/WebexAudioSettings/WebexAudioSettings.jsx

```diff
@@ -18,14 +18,21 @@ import WebexNoMedia from '../WebexNoMedia/WebexNoMedia';
 export default function WebexAudioSettings({className, meetingID, style}) {
   const cssClasses = webexComponentClasses('audio-settings', className);
   const [, display] = useMeetingControl('switch-microphone', meetingID);
+  // The browser api setSinkId() does not work properly on Firefox and Safari browsers.
```

Comment:
Thanks for adding the comment! Since it's relevant to the code here, let's keep the comment in the component